cesium/open.go

```diff
@@ -68,15 +68,7 @@ func Open(dirname string, opts ...Option) (*DB, error) {
 }
 
 func (db *DB) openVirtualOrUnary(ch Channel) error {
-	if db.closed.Load() {
-		return errDBClosed
-	}
-
-	db.mu.Lock()
-	defer db.mu.Unlock()
```

Comment:
We must not forget that this method is also called on db creation – but this is fine since we don't need a lock there as the db object is not exposed to the caller yet